style.css

```diff
@@ -557,17 +569,23 @@ footer {
     overflow: visible;
     flex-direction: row;
     flex-wrap: wrap;
+    margin-bottom: 20rem;
   }
 
   form {
     flex-direction: row;
     flex-wrap: wrap;
-    flex: 1 1;
+    flex: 3 3;
   }
 
   #contact-text {
-    flex: 1 1;
-    text-align: justify;
+    flex: 2 2;
+    text-align: left;
+    letter-spacing: normal;
+    vertical-align: middle;
+    line-height: 3rem;
+    padding: 2rem 0;
+    font-size: 1.5rem;
   }
```

Comment:
- [ ] Good job so far  👏  . You really put in a great effort, and I must commend your resilience. However, kindly follow the changes I'll specify below in line with the project requirements, and the last reviewer's comment 👍 

- [ ] In the input fields, you're missing the last name, and the email field is placed wrongly. I know you might see this to be normal, but the idea is to have you replicate the design as it is as much as possible just as it will be for a client or your manager 👍 
- [ ] Kindly add the above missing input field, and also adjust the form paragraph text accordingly 

### Your design:
![image](https://github.com/Movgto/my-portfolio/assets/50990925/7d4da8b4-b1c1-4d85-8ffc-c5395b78d865)

### Figma template:
![image](https://github.com/Movgto/my-portfolio/assets/50990925/3a3c491a-c2d3-475d-9b6c-7a62b70e2d95)

Reply:
Hi @Shedrack-Sunday, I know the last name field is missing, but the reason why is due to the note on the requirements that points out to ignore the last name field and only include a single field for the full name as you can see below:
![image](https://github.com/Movgto/my-portfolio/assets/107658605/51858168-d9c2-4993-af44-6de1bb7d2e12)
I would appreciate that you approve the project since it's my last review request, and I would rather not burn one of my extra reviews, thank you in advance 🙏 😞

Reply:
Hi @Movgto your request was noted and your project approved :100: :1st_place_medal: :+1: